azext_edge/edge/providers/edge_api/arccontainerstorage.py

```diff
@@ -10,3 +10,7 @@
 ARCCONTAINERSTORAGE_API_V1 = EdgeResourceApi(
     group="arccontainerstorage.azure.net", version="v1", moniker="arccontainerstorage"
```

Comment:
both these api groups exist at the same time?

Reply:
Interesting that docs show only usage of https://learn.microsoft.com/en-us/azure/azure-arc/container-storage/install-edge-volumes?tabs=multi

Reply:
yeah they both exist in the cluster

Reply:
we actually have another example with both api group exist, which is secret store:

secret-sync.x-k8s.io/v1alpha1
secrets-store.csi.x-k8s.io/v1

Reply:
Yea but those are both available on deployment with default settings.

Reply:
^-- I'm basically asking if containerstorage.azure.com is applicable to the azure arc version of container storage, as we are focused on azure arc services.